TestDrive.md

```diff
@@ -85,15 +85,28 @@ The original blog post also walks through creating a function:
 
 ## Add new functions to FaaS at runtime
 
-**Option 1** 
+**Option 1: Add Functions via docker-compose** 
 
 Edit the docker-compose stack file, then run ./deploy_stack.sh - this will only update changed/added services, not existing ones.
 
-**Option 2**
+**Option 2: Add Functions via faas UI**
 
-To attach a function at runtime you can use the "New function" button on the portal UI at http://localhost:8080/
+To attach a function at runtime you can use the "Create New Function" button on the portal UI at http://localhost:8080/ 
 
-**Option 3**
+<a href="https://pbs.twimg.com/media/C8opW3RW0AAc9Th.jpg:large"><img src="https://pbs.twimg.com/media/C8opW3RW0AAc9Th.jpg:large" width="600"></img></a>
+
+Creating a function via the UI:
+
+| Option                 | Usage             |
+|------------------------|--------------|
+| `Image`		 	| The name of the image you want to use for the function. A good starting point is functions/alpine |
+| `Service Name`  	 	| Describe the name of your service. No spaces please |
```

Comment:
> No spaces please

Might sound more professional to say what is valid for this field. The regex is: [a-zA-Z_0-9]